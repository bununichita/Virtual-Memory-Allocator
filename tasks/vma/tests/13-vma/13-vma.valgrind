==8910== Memcheck, a memory error detector
==8910== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==8910== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==8910== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==8910== Parent PID: 8909
==8910== 
==8910== 
==8910== HEAP SUMMARY:
==8910==     in use at exit: 0 bytes in 0 blocks
==8910==   total heap usage: 10 allocs, 10 frees, 8,377 bytes allocated
==8910== 
==8910== All heap blocks were freed -- no leaks are possible
==8910== 
==8910== For lists of detected and suppressed errors, rerun with: -s
==8910== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
