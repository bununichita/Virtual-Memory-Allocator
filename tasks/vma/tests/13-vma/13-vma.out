This zone was already allocated.
Total memory: 14 bytes
Free memory: 18 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 5 - 7
Miniblock 1:	5	-	7	| RW-
Block 1 end
